app/src/main/res/values/styles.xml

```diff
@@ -656,10 +658,10 @@
     <style name="RecentBookmarks.FaviconCard" parent="Mozac.Widgets.Favicon">
         <item name="android:layout_width">@dimen/recent_bookmark_item_width</item>
         <item name="android:layout_height">@dimen/recent_bookmark_item_height</item>
-        <item name="android:padding">@dimen/recent_bookmark_item_padding</item>
+        <item name="android:padding">16dp</item>
         <item name="cardBackgroundColor">?mozac_widget_favicon_background_color</item>
-        <item name="cardCornerRadius">@dimen/recent_bookmark_item_card_radius</item>
-        <item name="cardElevation">@dimen/recent_bookmark_item_card_elevation</item>
+        <item name="cardCornerRadius">8dp</item>
```

Comment:
I wish our styles had actual default values for this since we use them 8, 16, 24, etc everywhere.

Reply:
This is something we need to do in the future for clean-up that's tech debt we keep incurring. 🙁

Reply:
That's a good point. I wonder if there's something filed for that in our backlog... _writes myself a note to file this_